peer.js

```diff
@@ -410,6 +413,45 @@ TChannelPeer.prototype.connectTo = function connectTo() {
     return self.connect(true);
 };
 
+TChannelPeer.prototype.tryConnect = function tryConnect() {
+    var self = this;
+
+    var rightNow = Date.now();
```

Comment:
just rename now and rightNow to be more descriptive like "tryTime", "startTime" or whatever you want, but "rightNow" and "now" are no longer serving.